integration/helper_test.go

```diff
@@ -433,6 +434,116 @@ options:
 			Expect(err).To(HaveOccurred())
 			assertErrorsHandled()
 		})
+
+		It("skips batches if skip file discovered", func() {
+			// Run helper only with restore for a few batches and skip file defined
+			//
+			By("Write data file")
+			dataFile := dataFileFullPath
+			f, _ := os.Create(dataFile + ".gz")
+			gzipf := gzip.NewWriter(f)
+			// Named pipes can buffer, so we need to write more than the buffer size to trigger flush error
+			customData := "here is some data\n"
+			dataLength := 128*1024 + 1
+			customData += strings.Repeat("a", dataLength)
+			customData += "here is some data\n"
+
+			_, _ = gzipf.Write([]byte(customData))
+			_ = gzipf.Close()
+
+			// Write oid file
+			fOid, _ := os.Create(restoreOidFile)
```

Comment:
can we reuse the files created by `setupRestoreFiles()` instead of writing new code for file setup?